### PR TITLE
YLP-1504 Plug team alarms configuration to backend

### PR DESCRIPTION
**Pull request by TimCanu**

### CODE REVIEW

**Comment by TimCanu** [comment]
Hard to test with current auth0 situation. To check in the future

File: packages/yourloops/test/pages/hcp/patients-table.test.tsx [hunk 1]
```diff
@@ -122,13 +122,13 @@ describe("Patient list table", () => {
     });
   }
 
-  function renderPatientList(teamUsers: Patient[]) {
+  function renderPatientList(patients: Patient[]) {
```
Comment (yanngit) [inline packages/yourloops/test/pages/hcp/patients-table.test.tsx:125]:
```suggestion
  function renderPatientTable(patients: Patient[]) {
```

File: packages/yourloops/models/team.ts [hunk 2]
```diff
@@ -51,7 +51,8 @@ export interface ITeamMember {
   settings?: Settings | null;
   preferences?: Preferences | null;
   idVerified: boolean;
-  alert?: Alarm;
+  alarms?: Alarm;
```
Comment (yanngit) [inline packages/yourloops/models/team.ts:54]: maybe | null missing here ?

File: packages/yourloops/models/shoreline.ts [hunk 3]
```diff
@@ -96,7 +96,8 @@ interface IUser {
   preferences?: Preferences | null;
   /** Patient medical data. undefined means not fetched, null if the fetch failed */
   medicalData?: MedicalData | null;
-  alerts?: Alarm | null;
+  alarms?: Alarm | null;
+  monitoring?: { enabled: boolean}
```
Comment (yanngit) [inline packages/yourloops/models/shoreline.ts:100]: | null ?

File: packages/yourloops/test/common/index.ts [hunk 4]
```diff
@@ -320,5 +365,14 @@ export const patients: ITeamMember[] = [
...
     idVerified: false,
+    alarms : {
+      timeSpentAwayFromTargetRate: 10,
+      timeSpentAwayFromTargetActive: false,
+      frequencyOfSevereHypoglycemiaRate: 0,
+      frequencyOfSevereHypoglycemiaActive: false,
+      nonDataTransmissionRate: 0,
+      nonDataTransmissionActive: false,
+    },
+    monitoring: { enabled : false },
```
Comment (yanngit) [inline packages/yourloops/test/common/index.ts:376]: never set to true, is it on purpose ?

File: packages/yourloops/test/common/index.ts [hunk 5]
```diff
@@ -273,6 +273,15 @@ export const patients: ITeamMember[] = [
     profile: userPatient.profile,
     settings: userPatient.settings,
     idVerified: false,
+    alarms : {
+      timeSpentAwayFromTargetRate: 10,
```
Comment (yanngit) [inline packages/yourloops/test/common/index.ts:277]: define a const somewhere to reuse this code block ?

File: packages/yourloops/models/team.ts (hunk 2 above)
Comment (TimCanu) [reply packages/yourloops/models/team.ts:54]: I don't know anymore, if it is ok with you I'd like to keep it as is

File: packages/yourloops/models/shoreline.ts (hunk 3 above)
Comment (TimCanu) [reply packages/yourloops/models/shoreline.ts:100]: I don't know anymore, if it is ok with you I'd like to keep it as is

**Review by yanngit — APPROVED** [review]

**Pull request merged**